<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ordersvi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ordersvi.qrc">
    <normaloff>:/images/bad.svg</normaloff>:/images/bad.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLCDNumber" name="lcdAmount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="numDigits">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>60</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdSum">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Выручка:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdCash">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>100</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="numDigits">
     <number>7</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>293</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="Filemenu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="Servicemenu">
    <property name="title">
     <string>Сервис</string>
    </property>
   </widget>
   <widget class="QMenu" name="Helpmenu">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="Filemenu"/>
   <addaction name="Servicemenu"/>
   <addaction name="Helpmenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ordersvi.qrc"/>
 </resources>
 <connections/>
</ui>
